src/test/java/com/amazon/opendistroforelasticsearch/sql/intgtest/MainTestSuite.java

```diff
@@ -72,12 +72,10 @@
     HashJoinBasicTest.class,
     HashJoinMoreTest.class,
     ExplainTest.class,
-    WktToGeoJsonConverterTests.class,
```

Comment:
good change, I never updated this file, will do for all the ones I already migrated.